### PR TITLE
cleanups and minor improvements

### PR DESCRIPTION
**Pull request by muhomorr**

### CODE REVIEW

File: app/src/main/java/app/grapheneos/setupwizard/App.kt [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
 
 import android.app.Application
+import android.content.Context
 
 class App : Application() {
-    override fun onCreate() {
-        super.onCreate()
-        appContext = this
+
+    override fun attachBaseContext(base: Context) {
```
Comment (jiteshsingh) [inline app/src/main/java/app/grapheneos/setupwizard/App.kt:8]: is `attachBaseContext` more reliable and gets always called?

File: app/src/main/java/app/grapheneos/setupwizard/view/activity/LocationActivity.kt [hunk 2]
```diff
@@ -23,6 +24,9 @@ class LocationActivity : SetupWizardActivity(
 
     override fun setupActions() {
         enabled.setOnClickListener { LocationActions.setEnabled(enabled.isChecked) }
+        findViewById<TextView>(R.id.enabled_desc).setOnClickListener {
```
Comment (jiteshsingh) [inline app/src/main/java/app/grapheneos/setupwizard/view/activity/LocationActivity.kt]: in this case, better to attach listener on the container view (the linear layout)?
and also provide visual indications of the click?

pushed the change here; you can rebase. https://github.com/jiteshsingh/platform_packages_apps_SetupWizard/commits/location_click_container/

Comment (jiteshsingh) [reply app/src/main/java/app/grapheneos/setupwizard/view/activity/LocationActivity.kt]: or I can merge my change later, either way.

File: app/src/main/java/app/grapheneos/setupwizard/App.kt (hunk 1 above)
Comment (muhomorr) [reply app/src/main/java/app/grapheneos/setupwizard/App.kt:8]: Both onCreate() and attachBaseContext() are reliable, but Application.onCreate() is called after ContentProvider initialization.

**Pull request merged**